pythonforandroid/python.py

```diff
@@ -250,7 +250,7 @@ def build_arch(self, arch):
                 py_version = self.major_minor_version_string
                 if self.major_minor_version_string[0] == '3':
                     py_version += 'm'
-                shprint(sh.make, 'all',
+                shprint(sh.make, 'all', '-j',
```

Comment:
Don't we need to add the `cpu_count` as an argument for `-j` flag?

Reply:
As I wrote in my commit description, the number of jobs can be omitted in Python `make`, see for instance https://devguide.python.org/
But I will make it explicit if we prefer it more that way :+1: